src/assets/style-components/_reset.scss

```diff
@@ -29,10 +29,13 @@ dd {
     margin: 0;
 }
 
+html {
+    scroll-behavior: smooth;
```

Comment:
What is the difference between using this element here in `html` versus `body`?

Reply:
the smooth scrolling was not working when on the body, I read that the `html` element is the default scrolling element so I applied it to that to get the smooth scrolling working.